go/stellar/remote/remote.go

```diff
@@ -1115,15 +1115,19 @@ func IsAccountMobileOnly(ctx context.Context, g *libkb.GlobalContext, accountID
 
 // SetAccountMobileOnly will fetch the account bundle and flip the mobile-only switch,
 // then send the new account bundle revision to the server.
-func SetAccountMobileOnly(ctx context.Context, g *libkb.GlobalContext, accountID stellar1.AccountID) error {
+func SetAccountMobileOnly(ctx context.Context, g *libkb.GlobalContext, accountID stellar1.AccountID, enabled bool) error {
 	bundle, version, _, err := FetchAccountBundle(ctx, g, accountID)
 	if err != nil {
 		return err
 	}
 	if version == stellar1.BundleVersion_V1 {
 		return fmt.Errorf("mobile-only feature requires migration to v2 bundles")
 	}
-	err = acctbundle.MakeMobileOnly(bundle, accountID)
+	if enabled {
+		err = acctbundle.MakeMobileOnly(bundle, accountID)
+	} else {
+		err = acctbundle.MakeAllDevices(bundle, accountID)
```

Comment:
a few lines below here (in this file) is a method `MakeAccountAllDevices` which was intended to be the entry point to this behavior. can we get rid of it now? it might also be less work in the tests if you push this branching logic up a level to the caller. and then the caller can either come down to here or to `MakeAccountAllDevices`.

Reply:
I missed that, thanks. I'll take a look.

Reply:
So is this function changing?  I'd prefer SetAccountMobileOnly does the MakeMobileOnly.  And another function does MakeAllDevices instead of using a bool flag.